src/styles/_catalog.scss

```
@@ -1,9 +1,7 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
 
```

```suggestion
  grid-template-columns: repeat(1, 200px);
```
and add this by default and delete condition for max-width: 487px